module/Photo/view/photo/album-admin/index.phtml

```diff
@@ -17,25 +17,39 @@ $this->headLink()
             <div class="panel-body">
                 <ul class="tree">
                     <?php
-
                     function showTree($albums, $context)
                     {
-                        ?>
-                        <?php foreach ($albums as $album): ?>
-                        <li><a id="<?= $album->getId() ?>"
-                               href="<?= $context->url('admin_photo/album_index', array('album_id' => $album->getId())); ?>">
-                                <?= $album->getName() ?>
-                            </a>
-                            <?php if (count($album->getChildren()) > 0): ?>
-                                <ul>
-                                    <?= showTree($album->getChildren(), $context) ?>
-                                </ul>
-                            <?php endif ?>
-                        </li>
-                    <?php endforeach ?>
+                        foreach ($albums as $album) {
+                            echo sprintf(
+                                '<li><a id="%d" href="%s">%s</a>',
```

Comment:
Probeer alle html als html behouden, en de php in de html te injecteren, en niet andersom